client/src/js/actions.js

```diff
@@ -1,3 +1,12 @@
+export const joinGame = (gameId, name, id) => ({
+    type: 'joinGame',
+    payload: {
+        gameId,
+        name,
+        fbid: id
+    }
+});
```

Comment:
what will be the flow for joining a game? I was thinking about providing a URL that will be posted to someone chat